lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +199,40 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self._seed_urls = config.get_seed_urls()
+        self._config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        url = article_bs['href']
+        if isinstance(url, str):
```

Comment:
strange check

Reply:
According to mypy, `get` method returns `Union[str, List[str], None]` and fanction `_extract_url` should return a string